requirements.txt

```diff
@@ -1 +1,2 @@
-prefect>=2.0b9
\ No newline at end of file
+prefect>=2.0.0
+google-api-python-client>=2.20.0
```

Comment:
Directly embedding requirements in `setup.py` isn't a great practice. It does make some sense to match the collection template for now if that's the case though.

Reply:
We have other libraries in `setup.py` because they're extras. I'm ok with having this library in `requirement.txt` since it's more generalized compared to the service-specific client libraries.